src/components/ModalWelcome.native.tsx

```diff
@@ -141,16 +161,17 @@ const ModalWelcome = () => {
               }
               style={styles.helperActionButton}
             >
-              <Text style={styles.helperActionButtonText}>This ain’t my first rodeo. Skip it.</Text>
+              <GrayedText>This ain’t my first rodeo. Skip it.</GrayedText>
             </TouchableOpacity>
           }
+          <ThemedButton title='CHANGE THEME' onClick={() => toggleTheme()}></ThemedButton>
```

Comment:
The vision of **em** Settings is that they are edited like any other thought. So this is fine for demonstration purposes, but ultimately users will use existing idioms for accessing and editing Settings. We will eventually add a Settings button to the toolbar or footer that links to the `EM/Settings` context where the user can edit settings as thoughts.

Reply:
I understand. I did not polished these changes on purpose. It was just for demonstration purposes.